templates/static/scripts.js

```diff
@@ -66,8 +66,8 @@ document.getElementById('url-form').addEventListener('submit', async function(ev
 });
 
 async function downloadImage(imageUrl, text, headlineElement) {
-    const x = parseFloat(headlineElement.getAttribute('data-x')) || 0;
-    const y = parseFloat(headlineElement.getAttribute('data-y')) || 0;
+    const x = Math.round(parseFloat(headlineElement.getAttribute('data-x')) || 0);
```

Comment:
No need to do this rounding anymore

Reply:
Understood. I'll remove the rounding from the JavaScript code since the FastAPI endpoint now handles the conversion. I'll update the PR shortly.